src/js/Node.js

```diff
@@ -1779,7 +1779,8 @@ export class Node {
           this.dom.tdValue.parentNode.insertBefore(this.dom.tdColor, this.dom.tdValue)
 
           // this is a bit hacky, overriding the text color like this. find a nicer solution
-          this.dom.value.style.color = '#1A1A1A'
+          // #1A1A1A
+          this.dom.value.classList.add('jsoneditor-color-value')
```

Comment:
So far the code base uses utils function `addClassName` and `removeClassName` everywhere, for compatibility with older browsers. I just checked and using this in IE11 (the oldest still supported browser) using `classList.add` should be sort of safe. For consistency though it maybe best to simply keep using `addClassName` until we drop support for IE11.